src/views/AddItem.jsx

```diff
@@ -1,25 +1,44 @@
 import { useState } from 'react';
 import { addItem } from '../api/firebase.js';
 
-export function AddItem({ listToken }) {
+export function AddItem({ listToken, data }) {
 	const [itemName, setItemName] = useState('');
 	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(7);
-	const [itemAdded, setItemAdded] = useState(false);
 	const [error, setError] = useState(false);
+	const [userAlertMessage, setUserAlertMessage] = useState('');
 
-	// TODO: dynamically pass in the list token so item is added to correct list
 	const handleSubmit = async (e) => {
 		e.preventDefault();
+		const normalizedItemName = itemName.replace(/[\s\W]|_+/g, '').toLowerCase();
+
+		if (normalizedItemName === '') {
+			setUserAlertMessage('Please enter an item name.');
+			return;
```

Comment:
I'm curious, what is line 16 returning?

Reply:
In this case, nothing! It just means that the function will effectively stop running since the alert message has been set.